README.rst

```diff
@@ -1,13 +1,7 @@
 Overview
 ========
 
-**WARNING: As previously announced on our communication channels, the Kytos
-project will enter the "shutdown" phase on May 31, 2021. After this date,
-only critical patches (security and core bug fixes) will be accepted, and the
-project will be in "critical-only" mode for another six months (until November
-30, 2021). For more information visit the FAQ at <https://kytos.io/faq>. We'll
-have eternal gratitude to the entire community of developers and users that made
-the project so far.**
+This Network Application is part of the Kytos-NG project.
 
 |Experimental| |License| |Build| |Coverage| |Quality|
```

Comment:
Antonio, what you think about changing from Experimental to Stable in the tags above? I think mef_eline is already in a stable state.

Reply:
I'm really not sure. @viniarck @rmotitsuki what do you think?

Reply:
I agree to also remove `|Experimental|`, since it's going to land in prod and we'll also avoid breaking changes.